tests/interpreters/test_cf_interpreter.py

```diff
@@ -0,0 +1,69 @@
+from xdsl.builder import Builder, ImplicitBuilder
+from xdsl.dialects import arith, cf, func
+from xdsl.dialects.builtin import ModuleOp, i32
+from xdsl.interpreter import Interpreter
+from xdsl.interpreters.arith import ArithFunctions
+from xdsl.interpreters.cf import CfFunctions
+from xdsl.interpreters.func import FuncFunctions
+from xdsl.ir.core import Block, Region
+
+
+def triangle_fn(n: int) -> int:
```

Comment:
Where does this name vome from? Isn't this just the sum from 1 to n? 😅

Reply:
In my head it made sense :) it's the number of elements in a lower triangular matrix of size n. I'll rename to `sum_to`

Reply:
I see, that's true :sweat_smile: